src/olympia/reviewers/views.py

```diff
@@ -1005,6 +1005,7 @@ def review(request, addon, channel=None):
         form=form,
         has_versions_pending_rejection=has_versions_pending_rejection,
         is_admin=is_admin,
+        language_dict=dict(settings.LANGUAGES),
```

Comment:
Maybe also needed in `AbuseReportAdmin:addon_card()` and `abuse_reports()` views ? Those re-use (directly or indirectly) the template

Reply:
The use in the template is inside a `{% if name_translations %}` and we only add `name_translations` to the context in the reviewer view.